lib/init-transforms/outputPath.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (config, answer) => {
```

Comment:
I'd prefer `function` syntax everywhere besides inlined closures. In this case I think `function` is just more explicit.